src/app/zap-templates/templates/app/CHIPClusters.zapt

```diff
@@ -14,22 +15,23 @@
 namespace chip {
 namespace Controller {
 
-{{#chip_client_clusters}}
-class DLL_EXPORT {{asUpperCamelCase name}}Cluster : public ClusterBase
+{{/first}}
+class DLL_EXPORT {{asUpperCamelCase clusterName}}Cluster : public ClusterBase
 {
 public:
-    {{asUpperCamelCase name}}Cluster() : ClusterBase(app::Clusters::{{asUpperCamelCase name}}::Id) {}
-    ~{{asUpperCamelCase name}}Cluster() {}
-    {{#chip_cluster_commands}}
+    {{asUpperCamelCase clusterName}}Cluster() : ClusterBase(app::Clusters::{{asUpperCamelCase clusterName}}::Id) {}
+    ~{{asUpperCamelCase clusterName}}Cluster() {}
+    {{#all_outgoing_commands_for_cluster clusterName clusterSide isMfgSpecific="false"}}
```

Comment:
Why do we use "isMfgSpecific=false" ? It removes "MfgSpecificPing". At some point we want to support manufacturer specific commands.

Reply:
I could be wrong but I remember reading somewhere that mfg specific codes were not specifically used to denote that a cluster or command is mfg specific but the range of the cluster and command code is supposed to tell if it is mfg specific. Based on that assumption I turned this off since there will be no cluster or command code collisions with that assumption. However that is not the case for Zigbee 3.0

Reply:
Removed the isMfgSpecific flag. The MfgSpecificPing is being generated now